src/components/PokeForm.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+
+  const PokeForm = (props) => {
+
+    const [userChoice, setUserChoice] = useState("blank")
+
+    const handleChoice = (change) => {
+      setUserChoice(change.target.value)
+    }
+    return(
+      <>
+        <form onSubmit={(e) => props.getTypes(e, userChoice)}>
+          <select onChange={handleChoice} value={userChoice} id="pokemonType" name="pokemonType">
```

Comment:
IF I had to offer any suggestions. It would be organizing your keys on each line for better readability. That just comes down to preference in case you got into properties that had an abundance of keys.